src/gui/window_header.cpp

```diff
@@ -74,11 +74,14 @@ static const Rect16::Width_t icons_width(icon_usb_width + icon_lan_width);
 
 window_header_t::window_header_t(window_t *parent, string_view_utf8 txt)
     : window_frame_t(parent, GuiDefaults::RectHeader)
-    , icon_base(this, Rect16(rect.Left() + span, rect.Top(), rect.Width(), rect.Height()), 0)
-    , label(this, rect + Rect16::Left_t(span) - Rect16::Width_t(icons_width + span), is_multiline::no, is_closed_on_click_t::no, txt)
+    , icon_base(this, Rect16(0, rect.Top(), 40, rect.Height() - 5), 0)
```

Comment:
icon should be relative to header rect
, icon_base(this, Rect16(rect.Left(), rect.Top(), rect.Left()+40, rect.Height() - 5), 0)